live-build/misc/ansible-roles/appliance-build.virtualization-common/tasks/main.yml

```diff
@@ -45,3 +45,34 @@
       domain=
       dns1=
       dns2=
+
+- name: Add the CLI user with home directory
+  user:
+    name: cli
+    comment: Delphix CLI User
+    uid: 65434
+    non_unique: yes
+    group: staff
+    home: /export/home/cli
+    create_home: yes
+
+- name: Create directory where authorized keys will be specified for all CLI users
+  become_user: cli
+  file:
+    path: /export/home/cli/.ssh
+    state: directory
```

Comment:
Should we ensure that the directory has 0700 permissions? I seem to recall that these permissions are necessary for `sshd(8)` to be able to read the keys.

Reply:
Those restrictive permissions would be needed if we had an ssh key pair for the `cli`. In this case, we never really log in as `cli`. All we care about are the the `authorized_keys.*` files. I can try making this 0700 anyway, if it doesn't interfere with our management of the `authorized_keys.*` files.